proto/aquila/action_communication.proto

```diff
@@ -0,0 +1,65 @@
+syntax = "proto3";
+
+import "runtime_function_definition.proto";
+
+option ruby_package = "Tucana::Aquila";
+
+package aquila;
+
+// Event that gets admitted by an action
+message Event {
+  // Id of Event type
+  string event_type = 1;
+  // Payload (JSON) of event params
+  string payload = 2;
```

Comment:
If this should be a json object, this could instead be a [`google.protobuf.Struct`](https://protobuf.dev/reference/protobuf/google.protobuf/#struct) instead of a string that you need to parse manually